admin/tools/docker/latest/scripts/start.sh

```diff
@@ -30,4 +31,4 @@ set -e
 DIR=$(cd "$(dirname "$0")"; pwd -P)
 . $DIR/params.sh
 
-$QSERV_RUN_DIR/bin/qserv-start.sh
+$QSERV_RUN_DIR/bin/qserv-start.sh && tail -F $(find  $QSERV_RUN_DIR/var/log -type f)
```

Comment:
Yes, the goal is to print all logs to docker console (which isn't displayed by default).